### DISCUSSION TITLE
It would be nice to be able organize outlines into folders

### DISCUSSION THREAD

**Discussion by Tristram1** [vincode-io/Zavala#178] [Ideas]
As I use Zavala more... and thus Notes less... I am wishing that Zavala had folders to help me organize my growing number of outlines.

Notes allows only two levels of folders and I frequently run into that limitation because I have hundreds of notes.  It would be really great if Zavala allowed at least as many levels of folders as Notes does.

(thanks for a great product!)

**Comment by vincode-io** [comment]
This is being tracked as #173.
